string type
---
int8 result
